docs/rst/tutorials/core/qos/mutable/mutable.rst

```diff
@@ -0,0 +1,296 @@
+    colcon build
+
+
+Run
+---
+
+Open three terminals in the workspace folder.
+On each, you need to source Vulcanexus installation, as well as the package installation.
+Then, export the `FASTRTPS_DEFAULT_PROFILES_FILE` environment variable to point out to the corresponding profiles file.
+And run the ROS 2 node.
+
+First, in the first terminal, run the subscriber node, configured with the profiles1.xml file:
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+    cd ~/vulcanexus_ws
+    . install/setup.bash
+    export FASTRTPS_DEFAULT_PROFILES_FILE=./profiles1.xml
+    ros2 run vulcanexus_change_mutable_qos change_mutable_qos_sub
+
+
+Then, in another terminal, run the first publisher, configured also with the profiles1.xml file.
+This Publisher will then be configured with ownership strength value of 10.
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+    cd ~/vulcanexus_ws
+    . install/setup.bash
+    export FASTRTPS_DEFAULT_PROFILES_FILE=./profiles1.xml
+    ros2 run vulcanexus_change_mutable_qos change_mutable_qos_pub1
+
+
+At this point you will be able to see that both nodes are communicating, and the messages from Publisher 1 can be seen in the Subscriber.
+
+In the third terminal, run the second publisher, configured with the profiles2.xml file.
+This Publisher will then be configured with ownership strength value of 2.
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+    cd ~/vulcanexus_ws
+    . install/setup.bash
+    export FASTRTPS_DEFAULT_PROFILES_FILE=./profiles2.xml
+    ros2 run vulcanexus_change_mutable_qos change_mutable_qos_pub2
+
+
+This Publisher 2 starts sending messages (it can be seen that the number of the message starts from 1 while the messages from Publisher 1 are already in a higher number), and the Subscriber is still receiving messages from Publisher 1 and not from Publisher 2.
+This is because of the exclusive ownership.
+Publisher 1 has higher ownership strength than Publisher 2.
+
+.. raw:: html
+
+    <video width=100% height=auto autoplay loop controls muted>
+        <source src="../../../../../_static/resources/tutorials/core/qos/mutable/first_launch.mp4">
+        Your browser does not support the video tag.
+    </video>
+
+
+Change mutable QoS via command line
+-----------------------------------
+
+In this last section the ROS command `param set` will be used to change the value of the node's parameter created earlier.
+The parameter change will cause the parameter-change callback to be called, and then resulting in a change in the ownership strength.
+In another terminal, try the following code:
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+    cd ~/vulcanexus_ws
+    . install/setup.bash
+    ros2 param set /node2_change_mutable_qos pub2_ownership_strength 50
+
+
+With that execution, the ownership strength of the Publisher 2 has changed to become bigger than that of the Publisher 1.
```

Comment:
Please review this elsewhere. Bigger normally refers to physical size
```suggestion
With that execution, the ownership strength of the Publisher 2 has changed to become larger than that of the Publisher 1.
```

Reply:
Done